### DISCUSSION TITLE
Alert when seats open up on a specific flight

### DISCUSSION THREAD

**Discussion by zyonse** [jdholtz/auto-southwest-check-in#316] [General]
The direct flight for a route I usually take is currently sold out. Is there a way for this script to notify me if any space opens up?

**Comment by jdholtz** [comment]
No, the script currently doesn't do any look ups for any flights that you don't have currently booked, either through a confirmation number or your account. 

This feature may be more fitting for a separate project which deals with searching flights that are not already booked. Currently, this script pulls in flights the user has already booked and handles check-ins and low fare checks for only those flights.

**Reply by zyonse** [reply]
Makes sense. Thanks for the quick reply and for all the work you put into this project! Been using it for a couple years now.
